app/src/main/scala/uk/ac/wellcome/platformapi/exceptions/ElasticsearchExceptionMapper.scala

```diff
@@ -0,0 +1,22 @@
+package uk.ac.wellcome.platform.api.exceptions
+
+import org.elasticsearch.ElasticsearchException
+import com.twitter.finagle.http.{Request, Response}
+import com.twitter.finatra.http.response.ResponseBuilder
+import com.twitter.finatra.http.exceptions.ExceptionMapper
+import javax.inject.{Inject, Singleton}
+
+case class ErrorResponse(
+  message: String
+)
+
+@Singleton
+class ElasticsearchExceptionMapper @Inject()(response: ResponseBuilder)
+  extends ExceptionMapper[ElasticsearchException] {
+
+  override def toResponse(request: Request, exception: ElasticsearchException): Response = {
+    val errorResponse = ErrorResponse(exception.getMessage)
+
+    response.internalServerError.json(errorResponse)
```

Comment:
This exposes error messages from the Elasticsearch library to end-users. It would be better if this logged the error message, and then we just returned a generic 500 error.